crates/bevy_render/Cargo.toml

```diff
@@ -27,6 +28,8 @@ wgpu_trace = ["wgpu/trace"]
 ci_limits = []
 webgl = ["wgpu/webgl"]
 
+bevy_winit = ["bevy_winit/x11", "bevy_winit/wayland"]
```

Comment:
CI failures comes from this line, none of those features should be enabled here

Reply:
Thank you!  That fixed things.

Reply:
Now I remember why those were added:

![2023-03-22_16-39](https://user-images.githubusercontent.com/760949/227054256-6cdcc0da-9a70-45a3-a600-fba5e665cb79.png)